source/backend/discovery/test/getAllSdkResources.js

```diff
@@ -1547,6 +1549,74 @@ describe('getAllSdkResources', () => {
                     ]
                 });
             });
+            describe(AWS_DYNAMODB_STREAM, () => {
+
+                it('should discover DynamoDB Streams', async () => {
+                    const schema = require('./fixtures/additionalResources/dynamodb/stream.json');
+                    const {stream} = generate(schema);
+
+                    const mockDynamoDBStreamsClient = {
+                        createDynamoDBStreamsClient(accountId, credentials, region) {
+                            return {
+                                async getStreamInfo(streamArn) {
+                                    if(credentials.accessKeyId === ACCESS_KEY_X && region === EU_WEST_2) {
+                                        return [stream];
```

Comment:
Is this supposed to be wrapped in an array? I thought this function returned an object?

Reply:
I'm not sure! I was following the pattern of the other tests, strangely enough it passes with or without the array...